src/expressions/plyql.pegjs

```diff
@@ -343,6 +344,11 @@ ComparisonExpression
       return lhs[rest[1]](rest[3]);
     }
 
+  / IfNullToken OpenParen _ operand:Expression Comma _ fallbackValue:Expression CloseParen
+    {
+      return operand.fallback(fallbackValue);
+    }
```

Comment:
I think this should go with the function calls. This is a section for things that do not have OpenParen CloseParen